recipes/cherab_nagdis/recipe.yaml

```
@@ -0,0 +1,88 @@
+context:
+  name: cherab-nagdis
+  version: "0.1.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/munechika-koyo/cherab_nagdis/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 971aeab0e14aca7906654a5e614b62a2ab940a568c8352d8d009903dc8f5758f
+
+build:
+  skip:
+    - win
+    - match(python, "<3.10")
+  script:
+    - if: unix
+      then:
+        - export SETUPTOOLS_SCM_PRETEND_VERSION=$PKG_VERSION
+        - ${{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv .
+    - if: win
+      then:
+        - set SETUPTOOLS_SCM_PRETEND_VERSION=%PKG_VERSION%
+        - ${{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv .
```

```suggestion
  script: 
    content: ${{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv 
    env:
      SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }}
```
You should be able to simplify the build script to just these lines.